config/initializers/companies.yml

```diff
@@ -30,240 +7,268 @@
   address:    '128 Prinsep Street, Singapore 188655'
   hiring_url:
   email:      'hello@jollygoodcode.com'
-  contribution_count: 2
+  updated_at: '2017-01-01'
```

Comment:
Rails' naming convention for dates is suffix `_on`.

Reply:
fixed 👍